app/uk/gov/hmrc/cipphonenumberverification/services/VerifyHelper.scala

```diff
@@ -47,7 +54,7 @@ abstract class VerifyHelper @Inject()(passcodeService: PasscodeService, govUkCon
       passcodeService.persistPasscode(validatedPhoneNumber.phoneNumber) flatMap sendPasscode recover {
         case err =>
           logger.error(s"Database operation failed - ${err.getMessage}")
-          InternalServerError(Json.toJson(ErrorResponse("DATABASE_OPERATION_FAIL", "Database operation failed")))
+          InternalServerError(Json.toJson(ErrorResponse("PASSCODE_PERSISTING_FAIL", "Server has experienced an issue")))
```

Comment:
Theres quite a few of these code and message constants for the error responses now.  
Can we put the codes and messages in the one file please?  Call it something like "ErrorResponseConstants".
This is so that:

1. they are all in the one place which means the code is easier to maintain 
2. avoids hardcoding
3. the unit tests can use them as well as the normal code
4. when people add a new one they may find one they can reuse an existing one as they just look in that one file to see whats there already
5. it will help to make it easier to write the run book for production support, which we will have to do at some point.  As we can easily see all the error scenarios we have to document.
6. It will probably make refinement of new error tickets easier as well

Reply:
I have created an enumeration and updated the PR